### PR TITLE
Editorial updates to the v0.9.0 release notes and configuration file.

### PR DESCRIPTION
**Pull request by PahudPlus**

### CODE REVIEW

File: site/en/reference/milvus_config.md [hunk 1]
```diff
@@ -86,6 +86,7 @@ Before changing these settings, welcome to consult Milvus team on [GitHub issues
 | ------------------------ | ------------------------------------------------------------ | ------- | --------------- |
 | `primary_path`         | Primary directory used for the vector data files, index files, and the metadata. | Path   | `/var/lib/milvus`    |
 | `secondary_path` | A semicolon-separated list of secondary directories used vector data files and index files. Set this parameter when the data size is too much to fit in the `primary_path`. <br/>Each file, whether in `primary_path` or `secondary_path`, is assigned an equal part of the imported data.  Data Size per Directory = Total Data Size / Number of Directories. So make sure the available storage space in these files are enough. | Path   |     ` `       |
+| `file_cleanup_timeout` | The time gap between marking a file as `deleted` and physically deleting this file from disk. | Integer |     `10s`       |
```
Comment (cydrain) [inline site/en/reference/milvus_config.md:89]: need add "range [0, 3600]"

File: site/en/reference/milvus_config.md [hunk 2]
```diff
@@ -169,3 +170,20 @@ To switch to CPU-only mode, just set `enable` to `false`.
...
+| -------------------- | ------------------------------------------------------------ | ------------ | ------- |
+|  `trace.enable`      |   Whether to enable trace level logging in Milvus.           |    Boolean   |  `true` |
+|  `debug.enable`      |   Whether to enable debug level logging in Milvus.           |    Boolean   |  `true` |
+|  `info.enable`       |   Whether to enable info level logging in Milvus.            |    Boolean   |  `true` |
+|  `warning.enable`    |   Whether to enable warning level logging in Milvus.         |    Boolean   |  `true` |
+|  `error.enable`      |   Whether to enable error level logging in Milvus.           |    Boolean   |  `true` |
+|  `fatal.enable`      |   Whether to enable fatal level logging in Milvus.           |    Boolean   |  `true` |
+|  `path`              |  Absolute path to the folder holding the log files.  |    String    |   ` `   |
+|  `max_log_file_size` |  The maximum size of each log file.              |    Integer   | `256 (MB)` |
+|  `logrotate`         | The maximum number of log files that Milvus keeps for each logging level. |    Integer   | `10`  |
```
Comment (cydrain) [inline site/en/reference/milvus_config.md:188]: rename "logrotate" to "log_rotate_num"

File: site/en/reference/milvus_config.md [hunk 3]
```diff
@@ -169,3 +170,20 @@ To switch to CPU-only mode, just set `enable` to `false`.
...
+| Parameter            | Description                                                  | Type         | Default |
+| -------------------- | ------------------------------------------------------------ | ------------ | ------- |
+|  `trace.enable`      |   Whether to enable trace level logging in Milvus.           |    Boolean   |  `true` |
+|  `debug.enable`      |   Whether to enable debug level logging in Milvus.           |    Boolean   |  `true` |
+|  `info.enable`       |   Whether to enable info level logging in Milvus.            |    Boolean   |  `true` |
+|  `warning.enable`    |   Whether to enable warning level logging in Milvus.         |    Boolean   |  `true` |
+|  `error.enable`      |   Whether to enable error level logging in Milvus.           |    Boolean   |  `true` |
+|  `fatal.enable`      |   Whether to enable fatal level logging in Milvus.           |    Boolean   |  `true` |
+|  `path`              |  Absolute path to the folder holding the log files.  |    String    |   ` `   |
+|  `max_log_file_size` |  The maximum size of each log file.              |    Integer   | `256 (MB)` |
```
Comment (JinHai-CN) [inline site/en/reference/milvus_config.md:187]: range (64, 512)

File: site/en/reference/milvus_config.md (hunk 2 above)
Comment (JinHai-CN) [reply site/en/reference/milvus_config.md:188]: range (1, 4096)

File: site/en/reference/milvus_config.md [hunk 4]
```diff
@@ -169,3 +170,20 @@ To switch to CPU-only mode, just set `enable` to `false`.
...
 |  `wal_path`             |  Location of WAL log files.                                                            |    String          |    ` `     |
 </div>
+
+### Section `logs`
+
+<div class="table-wrapper" markdown="block">
+
+| Parameter            | Description                                                  | Type         | Default |
+| -------------------- | ------------------------------------------------------------ | ------------ | ------- |
+|  `trace.enable`      |   Whether to enable trace level logging in Milvus.           |    Boolean   |  `true` |
```
Comment (JinHai-CN) [inline site/en/reference/milvus_config.md:180]: 'Enable trace level logging' is OK.

File: site/en/reference/milvus_config.md (hunk 1 above)
Comment (JinHai-CN) [reply site/en/reference/milvus_config.md:89]: use (0, 3600)

File: site/en/release/release_notes.md [hunk 5]
```diff
@@ -6,6 +6,180 @@ sidebar_label: Release Notes
...
+
+| Milvus version | Python SDK version | Java SDK version | Go SDK version |
+| -------------- | ------------------ | ---------------- | -------------- |
+| 0.9.0          | 0.2.11             | 0.8.0            | 0.4.0          |
+
+#### New features
+
+
+- Checks the CPU instruction set, GPU driver version, and CUDA version, when Milvus starts up. [#2054](https://github.com/milvus-io/milvus/issues/2054) [#2111](https://github.com/milvus-io/milvus/issues/2111)
+- Adds a lock file to avoid concurrent access to the database by multiple Milvus instances. [#2059](https://github.com/milvus-io/milvus/issues/2059)
```
Comment (JinHai-CN) [inline site/en/release/release_notes.md:23]: Avoid multiple Milvus instances using the same data storage location.

File: site/en/release/release_notes.md [hunk 6]
```diff
@@ -6,6 +6,180 @@ sidebar_label: Release Notes
...
+|                     | C++               | Python               | Java              | Go                |
+| ------------------- | ----------------- | -------------------- | ----------------- | ----------------- |
+| Earlier than v0.9.0 | `GetIDsInSegment` | `get_vector_ids`     | `getVectorIds`    | `GetEntityIDs`    |
+| v0.9.0              | `ListIDInSegment` | `list_id_in_setment` | `listIDInSegment` | `ListIDInSegment` |
+
+
+
+|                     | C++          | Python          | Java          | Go           |
+| ------------------- | ------------ | --------------- | ------------- | ------------ |
+| Earlier than v0.9.0 | `SearchByID` | `search_by_ids` | `searchByIds` | `searchByID` |
```
Comment (JinHai-CN) [inline site/en/release/release_notes.md:122]: We never have this API before. so remove this item.

File: site/en/reference/milvus_config.md (hunk 3 above)
Comment (JinHai-CN) [reply site/en/reference/milvus_config.md:187]: 0 means no limitation and logroate not works.

File: site/en/reference/milvus_config.md [hunk 7]
```diff
@@ -64,6 +63,7 @@ Before changing these settings, welcome to consult Milvus team on [GitHub issues
 | `port`      | Port that Milvus server monitors. Port range: (1024, 65535). | Integer | `19530`    |
 | `deploy_mode` | Milvus deployment type. Options are `single`, `cluster_readonly` and `cluster_writable`. | DeployMode | `single`   |
 | `time_zone` | Use the UTC-x or UTC+x to specify a time zone. For example, use `UTC+8` for China Standard Time. | Timezone | `UTC+8`  |
+| `web_enable` |  Enable web server or not. | Boolean |  `true` |
```
Comment (JinHai-CN) [inline site/en/reference/milvus_config.md:66]: Whethere to ?

File: site/en/release/release_notes.md [hunk 8]
```diff
@@ -6,6 +6,180 @@ sidebar_label: Release Notes
...
+| v0.9.0              | `Flush`           | *UNCHANGED* | *UNCHANGED*              | *UNCHANGED* |
+
+
+
+
+
+|                     | C++                 | Python      | Java                         | Go          |
+| ------------------- | ------------------- | ----------- | ---------------------------- | ----------- |
+| Earlier than v0.9.0 | `CompactCollection` | `compact`   | `compact` and `compactAsync` | `Compact`   |
+| v0.9.0              | `Compact`           | *UNCHANGED* | *UNCHANGED*                  | *UNCHANGED* |
```
Comment (scsven) [inline site/en/release/release_notes.md:168]: CompactCollection and Compact in C++ SDK has been merge as Compact.

File: site/en/release/release_notes.md [hunk 9]
```diff
@@ -6,6 +6,180 @@ sidebar_label: Release Notes
...
+| v0.9.0              | `LoadCollection`    | `load_collection`    | `loadCollection`    | `LoadCollection`    |
+
+
+
+
+
+|                     | C++               | Python      | Java                     | Go          |
+| ------------------- | ----------------- | ----------- | ------------------------ | ----------- |
+| Earlier than v0.9.0 | `FlushCollection` | `flush`     | `flush` and `flushAsync` | `Flush`     |
+| v0.9.0              | `Flush`           | *UNCHANGED* | *UNCHANGED*              | *UNCHANGED* |
```
Comment (scsven) [inline site/en/release/release_notes.md:159]: FlushCollection and Flush merge into Flush in C++ SDK.

File: site/en/release/release_notes.md [hunk 10]
```diff
@@ -6,6 +6,180 @@ sidebar_label: Release Notes
...
+| v0.9.0              | `GetEntityByID`   | `get_entity_by_id`   | `getEntityByID`   | `GetEntityByID`  |
+
+
+
+
+
+|                     | C++               | Python               | Java              | Go                |
+| ------------------- | ----------------- | -------------------- | ----------------- | ----------------- |
+| Earlier than v0.9.0 | `GetIDsInSegment` | `get_vector_ids`     | `getVectorIds`    | `GetEntityIDs`    |
+| v0.9.0              | `ListIDInSegment` | `list_id_in_setment` | `listIDInSegment` | `ListIDInSegment` |
```
Comment (BossZou) [inline site/en/release/release_notes.md:116]: typo, should be list_id_in_segment

File: site/en/release/release_notes.md [hunk 11]
```diff
@@ -6,6 +6,180 @@ sidebar_label: Release Notes
...
+
+> See [CHANGELOG](https://github.com/milvus-io/milvus/blob/master/CHANGELOG.md) for more information.
+
+#### API changes
+
+##### Added methods
+
+
+
+| C++            | Python             | Java                 | Go |
```
Comment (BossZou) [inline site/en/release/release_notes.md:46]: API `connect`, `connected`, `disconnect` have been deprecated and not supported in SDK version 0.2.11. There may be a need to mention it.

Comment (PahudPlus) [reply site/en/release/release_notes.md:46]: The C++ SDK still keeps these three methods.

Comment (PahudPlus) [reply site/en/release/release_notes.md:46]: updated.

File: site/en/reference/milvus_config.md (hunk 7 above)
Comment (PahudPlus) [reply site/en/reference/milvus_config.md:66]: Done.

File: site/en/release/release_notes.md (hunk 5 above)
Comment (PahudPlus) [reply site/en/release/release_notes.md:23]: Prevents multiple Milvus instances from accessing the same Milvus database at the same time.

File: site/en/reference/milvus_config.md (hunk 4 above)
Comment (PahudPlus) [reply site/en/reference/milvus_config.md:180]: "Whether to" is required for a Boolean value.

File: site/en/reference/milvus_config.md (hunk 3 above)
Comment (PahudPlus) [reply site/en/reference/milvus_config.md:187]: [512, 4096]

File: site/zh-CN/reference/milvus_config.md [hunk 12]
```diff
@@ -62,10 +62,11 @@ $ docker restart <container id>
 | 参数        | 说明                                                         | 类型    | 默认值    |
 | ----------- | ------------------------------------------------------------ | ------- | --------- |
 | `address`   | Milvus 服务端监听的 IP 地址。                                  | String | `0.0.0.0`  |
-| `port`      | Milvus 服务端监听的端口号，范围：(1024, 65535)。               | Integer | `19530`   |
+| `port`      | Milvus 服务端监听的端口号，范围：[1024, 65535]。             | Integer | `19530`   |
```
Comment (cydrain) [inline site/zh-CN/reference/milvus_config.md]: 1024 and 65535 are excluded, so should use (1024, 65535)

File: site/zh-CN/reference/milvus_config.md [hunk 13]
```diff
@@ -62,10 +62,11 @@ $ docker restart <container id>
...
 | `deploy_mode` | Milvus 部署类型。选择 `single` ，`cluster_readonly` 或 `cluster_writable`。 | DeployMode | `single`  |
 | `time_zone` | 使用 UTC-x 或 UTC+x 来指定时区。比如，您可以使用 `UTC+8` 来代表中国标准时间。 | Timezone | `UTC+8`   |
-| `web_port` | Milvus 网络服务端监听的端口号，范围：(1024, 65535)。您可以使用 Milvus 网络服务端与 [Milvus RESTful API](sdk.md) 通信。| Integer | `19121`  |
+| `web_enable` | 选择是否启用 Web 服务器。 | Boolean | `true` |
+| `web_port` | Milvus 网络服务端监听的端口号，范围：[1024, 65535]。您可以使用 Milvus 网络服务端与 [Milvus RESTful API](sdk.md) 通信。 | Integer | `19121`  |
```
Comment (cydrain) [inline site/zh-CN/reference/milvus_config.md]: 1024 and 65535 are excluded, so should use (1024, 65535)

**Pull request merged**